test/explain/algorithm/test_gnn_explainer.py

```diff
@@ -172,10 +172,18 @@ def check_explanation(
         assert explanation.node_features_mask.shape == x.shape
         assert explanation.node_features_mask.min() >= 0
         assert explanation.node_features_mask.max() <= 1
+        assert explanation.node_features_mask[
+            0] != explanation.node_features_mask[1]
```

Comment:
I'm not sure I understand, you wanna make sure the node features are not the same for all the nodes ?